components/footer/footer-group.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import useTheme from '../use-theme'
+import { useFooterContext } from './footer-context'
+
+interface Props {
+  title: string | React.ReactNode
+}
+
+const defaultProps = {
+  title: '',
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterGroupProps = Props & typeof defaultProps & NativeAttrs
+
+const FooterGroup: React.FC<React.PropsWithChildren<FooterGroupProps>> = ({
+  title,
+  children,
+  ...props
+}) => {
+  const theme = useTheme()
+  const { breakPoint } = useFooterContext()
+  const [expand, setExpand] = React.useState(false)
+  return (
+    <div {...props}>
+      <h3 onClick={() => setExpand(!expand)}>{title}</h3>
+      <ul>{children}</ul>
+
+      <style jsx>{`
+        div{
+          margin-bottom:20px;
```

Comment:
It is recommended to use layout for layout.

Reply:
what does this mean exactly?